external_snaps_tests/__main__.py

```diff
@@ -68,14 +74,27 @@ def _is_git(repo):
 
 def _git_clone(url, repo_branch=None):
     temp_dir = tempfile.mkdtemp(prefix='snapcraft-')
-    command = ['git', 'clone', url, temp_dir]
+    command = ['git', 'clone', '--progress', url, temp_dir]
```

Comment:
Why `--progress` here? Is the git progress not otherwise reported in a subprocess exception?